pages_code.py

```diff
@@ -0,0 +1,4 @@
+def pages(n,m):
```

Comment:
`_code` is a given, this is a `.py` file.  Can this be renamed `pages.py` same for test file.